test/e2e/performanceprofile/functests/12_hypershift/hypershift.go

```diff
@@ -0,0 +1,200 @@
+package __hypershift
+
+import (
+	"context"
+	"fmt"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/utils/pointer"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	performancev2 "github.com/openshift/cluster-node-tuning-operator/pkg/apis/performanceprofile/v2"
+	manifestsutil "github.com/openshift/cluster-node-tuning-operator/pkg/util"
+	testutils "github.com/openshift/cluster-node-tuning-operator/test/e2e/performanceprofile/functests/utils"
+	testclient "github.com/openshift/cluster-node-tuning-operator/test/e2e/performanceprofile/functests/utils/client"
+	"github.com/openshift/cluster-node-tuning-operator/test/e2e/performanceprofile/functests/utils/discovery"
+	"github.com/openshift/cluster-node-tuning-operator/test/e2e/performanceprofile/functests/utils/hypershift"
+	testlog "github.com/openshift/cluster-node-tuning-operator/test/e2e/performanceprofile/functests/utils/log"
+	"github.com/openshift/cluster-node-tuning-operator/test/e2e/performanceprofile/functests/utils/nodepools"
+	"github.com/openshift/cluster-node-tuning-operator/test/e2e/performanceprofile/functests/utils/nodes"
+	"github.com/openshift/cluster-node-tuning-operator/test/e2e/performanceprofile/functests/utils/profiles"
+	hypershiftv1beta1 "github.com/openshift/hypershift/api/hypershift/v1beta1"
+	kubeletconfigv1beta1 "k8s.io/kubelet/config/v1beta1"
+)
+
+var _ = Describe("Multiple performance profile in hypershift", Ordered, func() {
+	var profile, secondProfile *performancev2.PerformanceProfile
+	var nodePools []*hypershiftv1beta1.NodePool
+	var np *hypershiftv1beta1.NodePool
+	var workerRTNodes []corev1.Node
+	var err error
+
+	nodeLabel := testutils.NodeSelectorLabels
```

Comment:
I don't see other use outside BeforeAll, let's move it below close to its use

Reply:
Updated with latest commit.